code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -899,28 +899,69 @@
 
 /datum/reagent/medicine/earthsblood //Created by ambrosia gaia plants
 	name = "Earthsblood"
-	description = "Ichor from an extremely powerful plant. Great for restoring wounds, but it's a little heavy on the brain."
+	description = "Ichor from an extremely powerful plant. Some conspiracy theorists INSISTS this plant was used by Centcom during Plasmastock to minimize any chance of rioting."
 	color = rgb(255, 175, 0)
 	overdose_threshold = 25
+	taste_description = "shag carpeting and rock"
+	metabolization_rate = 0.5
+	var/ODcycle = 0 //adds to current_cycle in calculation. 5 cycles while OD = 1 "normal" cycle (basically you're getting 1.2 instead of 2 cycles during OD)
 
 /datum/reagent/medicine/earthsblood/on_mob_life(mob/living/carbon/M)
-	M.adjustBruteLoss(-3 * REM, 0)
-	M.adjustFireLoss(-3 * REM, 0)
-	M.adjustOxyLoss(-15 * REM, 0)
-	M.adjustToxLoss(-3 * REM, 0)
-	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 2 * REM, 150) //This does, after all, come from ambrosia, and the most powerful ambrosia in existence, at that!
-	M.adjustCloneLoss(-1 * REM, 0)
-	M.adjustStaminaLoss(-30 * REM, 0)
-	M.jitteriness = min(max(0, M.jitteriness + 3), 30)
+	if(current_cycle <= 25) //5u has to be processed before u get into THE FUN ZONE
```

Comment:
earthsblood has a metabolization rate of 0.4u/tick; you should replace the "5u" in this comment with a "10u"

Reply:
It doesn't. Earthblood has a metabolization_rate of 0.5, meaning the correct number for this comment is 12.5u
:rainbow: The more you know :rainbow:

Reply:
The correct number would be 12.5